docs/modules/develop/pages/guide_development_app.adoc

```diff
@@ -18,18 +17,15 @@ sudo -u postgres psql -c "CREATE USER decidim_development_app_user WITH SUPERUSE
 [start=2]
 . Save the database configuration in your environment variables configuration. If you've followed the xref:install:manual.adoc[Decidim installation manual],
 then you should have a working https://github.com/rbenv/rbenv-vars[rbenv-vars] in your environment:
-
 [source,console]
 ----
 echo "DATABASE_USERNAME=decidim_development_app_user" >> ~/.rbenv-vars
 echo "DATABASE_PASSWORD=changeme" >> ~/.rbenv-vars
 ----
-
```

Comment:
Can you re-add the line break here. Sorry, the GitHub suggestion tool does not allow making such suggestion.